src/surbhi/WordCount.java

```diff
@@ -0,0 +1,32 @@
+package surbhi;
+public class WordCount {
+
+	void wordFrequencyCount(String string) {
+		System.out.println("Given string: " + string);
+		int max = 0, maxWordIndex = 0;
+		String[] strArray = string.split(" ");
+		for (int index = 0; index < strArray.length; index++) {
+			String word = strArray[index];
+			int counter = 0;
+			if (string.contains(word)) {
```

Comment:
Here contains method makes no sense, as you checking "word" in "string" (from the string you get this keyword so definitely that key word is in "string"), just logically correction